x-pack/plugin/ent-search/qa/rest/roles.yml

```diff
@@ -6,36 +6,19 @@ admin:
     - monitor
   indices:
     - names: [
-        # indices
-        "test-index",
-        "test-index1",
-        "test-index2",
-        "test-index3",
-        "test-index4",
-        "test-index-does-not-exist",
-        "test-search-index1",
-        "test-search-index2",
-        # Search Applications (needed to create aliases)
-        "test-search-application",
-        "test-search-application-1",
-        "test-search-application-2",
-        "test-search-application-with-list",
-        "another-test-search-application",
-        "test-updated-search-application",
-        "test-error-search-application",
-        "test-re-creating-search-application",
-        "test-search-application-to-delete",
-        "test-nonexistent-search-application",
-        "test-search-application-with-invalid-template"
+        # indices and search applications
+        "test-*",
+        "another-test-search-application"
     ]
-      privileges: [ "manage", "write" ]
+      privileges: [ "manage", "write", "read" ]
```

Comment:
Why is it needed (the `read` priv)?

Reply:
This was required to get rule tests to pass.